lib/connextclient/ConnextClient.ts

```diff
@@ -259,9 +260,27 @@ class ConnextClient extends SwapClient {
     return this._totalOutboundAmount.get(currency) || 0;
   }
 
-  public maxChannelInboundAmount = (_currency: string): number => {
-    // assume MAX_AMOUNT since Connext will re-collaterize accordingly
-    return MAX_AMOUNT;
+  public checkInboundCapacity = (inboundAmount: number, currency: string) => {
```

Comment:
Would be nice to get an integration test (or two) for this as there's quite a bit of logic in this public function.